### PR TITLE
Turn on SiP for 1990

### PR DESCRIPTION
**Pull request by cehsu**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4475

### CODE REVIEW

File: src/js/edu-benefits/1990-rjsf/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -61,26 +65,31 @@ class IntroductionPage extends React.Component {
...
-              afterText="»"/>
-          </div>
-        </div>
-        <div className="omb-info--container">
+        <SaveInProgressIntro
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}>
+          Complete the form before submitting to apply for education benefits with the 22-1990.
```
Comment (cehsu) [inline src/js/edu-benefits/1990-rjsf/components/IntroductionPage.jsx]: It looks like our language here and in the HCA form differ, should we confirm the content here?

Comment (cvalarida) [reply src/js/edu-benefits/1990-rjsf/components/IntroductionPage.jsx]: Great point. We should probably just keep it like this for now and make another ticket for Peggy to go over in more detail.

Comment (jbalboni) [reply src/js/edu-benefits/1990-rjsf/components/IntroductionPage.jsx]: According to @peggygannon, we should use the simpler "Please complete the XXXX form to apply for education benefits" form. So we can switch to that here (I updated the 1990e already).

**Review by jbalboni — COMMENTED** [review]
Looks good so far. There's also a `usa-alert` in the subway map, that should be removed (along with the `<br/>` tag after it.

**Comment by cehsu** [comment]
Thanks @jbalboni, changes made.

File: src/js/pensions/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -80,7 +80,7 @@ class IntroductionPage extends React.Component {
           pageList={this.props.route.pageList}
           {...this.props.saveInProgressActions}
           {...this.props.saveInProgress}>
-          Complete the form before submitting to apply for pension benefits with the 21-527EZ.
+          Please complete the 21-527EZ form to apply for pension benefits.
```
Comment (cehsu) [inline src/js/pensions/components/IntroductionPage.jsx]: @jbalboni is it alright to update this to match the other introduction pages?

Comment (jbalboni) [reply src/js/pensions/components/IntroductionPage.jsx]: Yep, go for it.

Comment (jbalboni) [reply src/js/pensions/components/IntroductionPage.jsx]: Also, can you update the 5495 one to have the `22-` in front of the form number?

**Review by jbalboni — APPROVED** [review]
This looks good, but let's wait to merge it until Monday, so we have a few days with the 1990 in production and can fix any issues without a form data migration.

File: src/js/edu-benefits/5495/components/IntroductionPage.jsx [hunk 3]
```diff
@@ -59,7 +59,7 @@ class IntroductionPage extends React.Component {
           messages={this.props.route.formConfig.savedFormMessages}
           {...this.props.saveInProgressActions}
           {...this.props.saveInProgress}>
-          Please complete the 5495 form to apply to manage your education benefits.
+          Please complete the 22-5495 form to apply to manage your education benefits.
```
Comment (cehsu) [inline src/js/edu-benefits/5495/components/IntroductionPage.jsx]: @jbalboni, ask, and ye shall receive.

**Comment by jbalboni** [comment]
@cehsu Feel free to merge this now.

**Pull request merged**